### DISCUSSION TITLE
Is there any way to disable git feature ?

### DISCUSSION THREAD

**Discussion by y4code** [glushchenko/fsnotes#1398] [Q&A]
I just want to use iCloud to sync all notes, and do not need the history feature, because every time i press CMD+s, it alerts 'git push error', the git feature settings is hard to understand.

**Comment by y4code** [comment]
![截屏2022-09-26 16 36 13](https://user-images.githubusercontent.com/8910420/192239908-01a67433-8f76-4f7a-8495-6b70a0b8aa66.png)

**Comment by y4code** [comment]
sorry, just found that libgit2 is a new library added to fsnotes

[FSNotes ](https://twitter.com/FSNotesApp)🇺🇦
[@FSNotesApp](https://twitter.com/FSNotesApp)
Good news guys! I managed to build libgit2, which means that now it will be possible not only to store history in git, but also to push! Will be available in next beta ;)

**Reply by glushchenko** [reply]
Hi, this is related to beta, and will be fixed in beta 5.

**Comment by glushchenko** [comment]
Fixed in release
